lisp/cues/media_cue.py

```diff
@@ -69,6 +69,11 @@ def __start__(self, fade=False):
         if fade:
             self._on_start_fade()
 
+        # If there was a fade-down, but there wasn't a fade-up - either
```

Comment:
It should be better to keep the `fade-in` and `fade-out` terms, as in other parts of code-base. (also in the line below)

Reply:
Hmm, sorry. Mentally I think of volumes going up or down, rather than in or out.

Hope the new comment's better.

Reply:
that make sense, but, at least in my experience, `fade-in/fade-out` is what is most commonly used :-)